releasenotes/notes/envoy-eds-cache.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+releaseNotes:
+- |
+  **Enabled** the configuration to use endpoint cache at Envoy when there is a delay
```

Comment:
How can this be disabled @ramaraochavali ?

Reply:
Setting it to "" in proxy config runtime values annotation seems to be the way - probably worth documenting here.